tests/runtime/test_runtime_metrics_api.py

```diff
@@ -43,3 +43,27 @@ def test_manually_start_runtime_metrics(run_python_code_in_subprocess):
     )
 
     assert status == 0
+
+
+def test_start_runtime_metrics_via_env_var(ddtrace_run_python_code_in_subprocess):
+    """
+    When running with ddtrace-run and DD_RUNTIME_METRICS_ENABLED is set
+        Runtime metrics worker starts and there are no errors
+    """
+
+    _, _, status, _ = ddtrace_run_python_code_in_subprocess(
+        """
+from ddtrace.internal.runtime.runtime_metrics import RuntimeWorker
+assert RuntimeWorker._instance is None
```

Comment:
```suggestion
assert RuntimeWorker.status == ServiceStatus.STOPPED
```

we've been bitten by having our tests rely too much on implementation details in the past 😅

Reply:
Great suggestion but for this to work we'd need to implement `status` as a class method on `RuntimeWorker`, or as a static method on the public `RuntimeMetrics` with a new PR. WDYT?

Reply:
ahh right. Hmm I don't think it should block this PR but maybe it'd be nice to have in the future!